src/main/webapp/app/course/learning-paths/participate/learning-path-container.component.scss

```diff
@@ -0,0 +1,13 @@
+.main-container {
+    width: 100%;
+    margin-left: 0;
+}
+
+.course-info-bar {
+    margin: 0 !important;
+}
+
+.tab-bar-exercise-details {
+    margin-left: 0 !important;
+    margin-right: 0 !important;
+}
```

Comment:
Hmm ... I'd like to discuss this approach - seems like you're changing the style of different components here (outside of and not even contained by this component), right?
This may lead to confusion if someone just look at the code of the affected components